gradle.properties

```
@@ -1,5 +1,5 @@
-ltrVersion = 1.5.11
-elasticsearchVersion = 9.0.1
+ltrVersion = 1.5.13
```

Can you revert that back to 1.5.11 please?
If it's only a version bump without any changes to the code itself in terms of bug fixes, new features etc. we keep the `ltrVersion` stable and reflect this in the combination of `ltrVersion` and `elasticsearchVersion`.

```suggestion
ltrVersion = 1.5.11
```